templates/DatasetMonitoring/projects.html.twig

```diff
@@ -5,7 +5,7 @@
     {% endif %}
     {% if not pdf | default %}
         <a href="/html2pdf?url={{ url(app.request.get('_route'), {'id': id, 'renderer': 'html2pdf'}) }}&format=Letter&orientation=landscape&margin=1cm&download=true&filename={{pdfFilename}}" class="download-pdf" title="save as pdf">
```

Comment:
Is this supposed to be this way?

Reply:
Yes, this one is a bit odd. The html2pdf service is external, and not part on our code, or rather this repo.

Reply:
and it's not base_url aware. We have the node service on the server's /.